romana-install/roles/romana/postinstall/kubernetes/agent/tasks/romana_kube_agent.yml

```diff
@@ -5,7 +5,7 @@
     creates: /usr/local/lib/romana-kube-agent
   
 - name: Configure IP address for agent
-  replace: dest="/usr/local/lib/romana-kube-agent" regexp="192\.168\.0\.10" replace="{{ stack_nodes.Controller.mgmt_ip }}"
+  replace: dest="/usr/local/lib/romana-kube-agent" regexp="192\.168\.0\.10" replace="{{ romana_master_ip }}"
```

Comment:
I think once it's rewritten in Go, it'll either have a config file or command-line option that the installer can use without needing an in-place change of the file.